### PR TITLE
Add user notification when using VS 2015 < U3

### LINKED ISSUE #476: Display a message to users with VS2015 under update 3
## Description

Before dropping the support for older versions of Visual Studio (i.e. < update 3) we need to warn the users that this release will be the last for their version of VS and that they will need to update the version to U3 or VS 2017 to be able to benefit from the new releases.

## Motivations

Visual Studio 2015 under Update 3 are no longer supported by Microsoft. The support of these older versions forces us to rely on Roslyn 1.0 which has some bugs and less features than 1.3.

### PR DESCRIPTION
**Pull request by Evangelink**
Fix #476

Screenshot of the message bar:
2018-01-11_15h45_10

Screenshot of the output window:
2018-01-11_15h45_26

### CODE REVIEW

File: src/Integration/DeprecationManager.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                vsVersion < VisualStudio2015Update3Version)
+            {
+                WriteMessageToOutput();
+                ShowDeprecationBar();
+            }
+        }
+
+        private void WriteMessageToOutput()
+        {
+            const string message =
```
Comment (Evangelink) [inline src/Integration/DeprecationManager.cs]: Please feel free to suggest another message. FYI I haven't put any web link as we haven't defined it yet.

File: src/Integration/DeprecationManager.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                "***   Please update to Visual Studio 2015 Update 3 or Visual Studio 2017 to benefit   ***\r\n" +
+                "***   from new features.                                                              ***\r\n" +
+                "*****************************************************************************************";
+
+            sonarLintOutput.Write(message);
+        }
+
+        private void ShowDeprecationBar()
+        {
+            const string message = "Newer versions of SonarLint will not work with this version of Visual Studio.";
```
Comment (Evangelink) [inline src/Integration/DeprecationManager.cs]: Please also feel free to suggest changes to this message.

Comment (michalb-sonar) [reply src/Integration/DeprecationManager.cs]: The message does not say how to fix that. As a user, I would have thought this means I need to move to VS 2017.

Perhaps use the same message as in output (minus asterisks)?

**Comment by michalb-sonar** [comment]
@duncanp-sonar +1 I think users who do not want/cannot update is a small minority, so don't think we should worry about that case. And if there is a significiant push-back, we can tweak it later.

**Pull request merged**